source/includes/trip-api.md

```diff
@@ -755,6 +829,80 @@ curl -d '{ "optionId": "f0e34a1b-2b3d-4747-b426-292633b615b4", "pickUpAddressNot
       "description": "Piazzale Roma",
       "image": "https://daytrip.imgix.net/management/venice.png?w=480&q=50"
    },
+   "trip": {
+      "id":"519314e3-cd92-41e0-85b6-c652c345e9d0",
+      "type":"Private",
+      "englishSpeakingDriver": false,
+      "distanceKm":334,
+      "travelTimeMinutes":268,
+      "pickUp":{
+         "lat": 46.067648,
+         "lon": 7.775185,
+         "time":"2022-12-05T18:00:00Z",
+         "meetAndGreet": true
```

Comment:
This should be false, see above.

Reply:
updated in a9f6f73544469625f3951b112800b772e495a9d4